data/pre_gen_info.txt

```diff
@@ -4,8 +4,9 @@
 
     <include>
       <uri>model://ground_plane</uri>
+       <color>255 0 0</color>
```

Comment:
What's the red color for?

Reply:
Changed my bad. Anyways it doesn't reflect in simulation.